modules/travel_pay/spec/services/claims_service_spec.rb

```diff
@@ -134,6 +134,9 @@
     let(:user) { build(:user) }
     let(:claims_by_date_data) do
       {
+        'statusCode' => 200,
+        'message' => 'Data retrieved successfully.',
+        'success' => true,
```

Comment:
Should this be wrapped in a `metadata: {...}` hash?

Reply:
nope

Reply:
Nope. This mock data is the original response from the `client`; the service (what this test covers) is what transforms the `client` Faraday response status info into the `metadata: {...}` hash format for us to return to the appt > claim association code.